src/test/test_cases.py

```diff
@@ -0,0 +1,52 @@
+import pytest
```

Comment:
Maybe:
```suggestion
import pytest
from pytest.mark import parametrize
```

Shortens the decorator lines.